X16 Reference - 12 - Hardware.md

```diff
@@ -0,0 +1,133 @@
+## Chapter 12: Hardware Pinouts
+
+This chapter covers pinout for the I/O ports and performing firmware updates on the SMC, VERA, and system ROMs.
+
+### Port and Socket Listing
+* VERA Connectors
+  * VERA Mainboard socket
+  * SD Card
+  * Composite Video
+  * S-Video
+  * VGA
+* SNES Controller Ports (x2)
+* IEC Port
+* PS/2 Keyboard and mouse
+* Expansion Ports (x4)
+* User Port header
+* ATX Power Supply
+* Front Panel
+
+## Disclaimer
+
+The instructions and information in this document are the best available information at the time of writing. This information is subject to change, and no warranty is implied. We are not liable for damage or injury caused by use or misuse of this information, including damage caused by inaccurate information. Interfacing and modifying your Commander X16 is done solely at your own risk.
+
+If you attempt to upgrade your firmware and the process fails, one of our community members may be able to help. Please visit the forums or the Discord community, both of which can be reached through https://commanderx16.com.
+
+### VERA Connectors
+
+TODO
+
+### SNES Ports
+
+The computer contains two SNES style ports and will work with Super Nintendo compatible game pads. 
+
+<img src="images/SNES_Controller_Female.svg" alt-text="SNES Controller Port" width="256" />
+
+| Pin # | Description | Wire Color
+|-------|-------------|-------------
+| 1 | +5v | White
+| 2 | Data Clock | Yellow/Red
+| 3 | Data Latch | Orange
+| 4 | Serial Data | Red/Yellow
+| 5 | N/C | -
+| 6 | N/C | -
+| 7 | Ground | Brown
+
+The Data Clock and Data Latch are generated by the computer and can be shared between the two ports. The Serial Data line is unique per controller.
+
+Thanks to [Console Mods Wiki](https://consolemods.org/wiki/SNES:Connector_Pinouts)
+
+### IEC Port
+
+The IEC port is a female 6 pin DIN 45322 connector. The pinout and specifications are the same as the Commodore 128 computer, with the required lines for Fast IEC, as used by the 1571 and 1581 diskette drives. 1541 drives are also compatible, using standard IEC mode at 4800 bits/sec.
+
+<img src="images/iec_port.gif" alt-text="IEC Serial Port" />
```

Comment:
Should markdown be used for images instead of <img> tags?  This could be rewritten as:
```
![IEC Serial Port](images/iec_port.gif)
```

Reply:
As mentioned elsewhere, I have settled on using img tags exclusively because Markdown does not support the width field, and that is necessary to keep an even size on the images.